packages/webapp/components/TreasuryBalance.tsx

```diff
@@ -19,12 +19,40 @@ import { useGnarsV2TokenBalanceOf } from "utils/sdk"
 import { formatEther } from "viem"
 import { useBalance } from "wagmi"
 import { ShredIcon } from "./Icons"
-
+import { useState, useEffect } from "react";
+import axios from "axios";
+import { set } from "lodash"
```

Comment:
not getting used anywhere

Reply:
it is, I am using both
<img width="430" alt="image" src="https://github.com/gnars-dao/gnars-frontend/assets/116202536/b5562e74-3d3d-4efd-9986-2b7acd38f3fb">

Reply:
`set` is a function from the lodash library for setting values of an object. i dont think it needs to be imported to use `useState`

Reply:
thats from the old code, I removed , but that was not part of this addition, I must have deleted and redo . understood